src/store/transaction/transaction.thunks.js

```diff
@@ -292,14 +292,11 @@ function withdraw (amount, account, exit, completeDelayedWithdrawal, instantWith
 
     // Differentiate between a withdraw on the Hermez SC and the DelayedWithdrawal SC
     if (!completeDelayedWithdrawal) {
-      return Tx.withdraw(
-        amount,
-        account.accountIndex,
-        account.token,
-        wallet.publicKeyCompressedHex,
-        exit.batchNum,
-        exit.merkleProof.siblings,
+      return Tx.withdrawCircuit(
+        exit,
         instantWithdrawal,
+        'https://raw.githubusercontent.com/hermeznetwork/hermezjs/main/withdraw-circuit-files/withdraw.wasm',
```

Comment:
Maybe they can be constants?

Reply:
I thought about that especially because they are used in 2 modules. Would you declare them as global constants somewhere and import them here and in `service-worker.js` or do we do local declarations?

Reply:
I would use the constants file that we have

Reply:
Same here, we have a `constants.js` file under the `src` folder to keep this things easy to change and shareable between different modules.

Reply:
Importing the URLs is not working on the service worker. I've been reading about this and it looks like service workers do not fully support ES6 imports. So far I've moved the URLs to constants and used them on `transaction.thunks.js` but not on `service-worker.js`. I added a comment in constants to make sure we do update the URL's in the service worker when we update them in the constants.

Maybe you have more experience with service workers and can point to a solution that allows us to import the constants there?